### PR TITLE
209 reorder codelists

### LINKED ISSUE #209: Order codes in codelists
We should check that codes in each codelist are ordered logically. For example, in the [nodeStatus codelist](https://open-fibre-data-standard.readthedocs.io/en/latest/reference/codelists.html#nodestatus), it would make sense for 'inactive' to appear between 'underConstruction' and 'operational', rather than at the end of the list.

For most codelists, I think alphabetical order is fine, but we can use this issue to agree the appropriate order for each codelist.

### PR DESCRIPTION
**Pull request by odscjen**
**Related issues**

closes #209 

**Description**

**Merge checklist**

- [x] Update the changelog ([style guide](https://ofds-standard-development-handbook.readthedocs.io/en/latest/style/changelog_style_guide.html))
- [x] Run `./manage.py pre-commit` to update derivative schema files, reference documentation and examples

If there are changes to `network-schema.json`, `network-package-schema.json`, `reference/publication_formats/json.md`, `reference/publication_formats/geojson.md` or `guidance/publication.md#how-to-publish-large-networks`, update the relevant manually authored examples:

- [ ] `examples/json/`:
  - [ ] `network-package.json`
  - [ ] `api-response.json`
  - [ ] `multiple-networks.json`
  - [ ] `network-embedded.json`
  - [ ] `network-separate-endpoints.json`
  - [ ] `network-separate-files.json`
  - [ ] `nodes-endpoint.json`
  - [ ] `spans-endpoint.json`
- [ ] `examples/geojson/`:
  - [ ] `api-response.geojson`
  - [ ] `multiple-networks.geojson`

If you used a validation keyword, type or format that is not [already used in the schema](https://ofds-standard-development-handbook.readthedocs.io/en/latest/standard/schema.html#json-schema-usage):

- [ ] Update the list of validation keywords, types or formats in [JSON Schema usage](https://ofds-standard-development-handbook.readthedocs.io/en/latest/standard/schema.html#json-schema-usage).
- [ ] Add a field that fails validation against the new keyword, type or format to [`network-package-invalid.json`](https://github.com/Open-Telecoms-Data/open-fibre-data-standard/blob/0.1-dev/examples/json/network-package-invalid.json).
- [ ] Check that [OFDS CoVE](https://ofds.cove.opendataservices.coop/) reports an appropriate validation error.

If you added a normative rule that is not encoded in JSON Schema:

- [ ] Update the list of [other normative rules](https://ofds-standard-development-handbook.readthedocs.io/en/latest/standard/schema.html#other-normative-rules).
- [ ] Add a field that does not conform to the rule to [`network-package-additional-checks.json`](https://github.com/Open-Telecoms-Data/open-fibre-data-standard/blob/0.1-dev/examples/json/network-package-additional-checks.json).
- [ ] Open a [new issue](https://github.com/Open-Telecoms-Data/lib-cove-ofds/issues/new/choose) to add an additional check to Lib Cove OFDS.

If there are changes to `examples/geojson/nodes.geojson` or `examples/geojson/spans.geojson`, check and update the data use examples:

- [ ] `examples/leaflet/leaflet.ipynb`
- [ ] `examples/qgis/geojson.qgs`

### CODE REVIEW

**Comment by odscjen** [comment]
reporting broken link

> (guidance/publication: line  348) broken    https://linux.die.net/man/3/libuuid - 403 Client Error: Forbidden for url: https://linux.die.net/man/3/libuuid

but its not broken, following that url takes you to a working webpage, all be it one with an irritating popup

**Comment by odscjen** [comment]
It's also unclear why running `manage/py pre-commit` changed 2 example files when all this PR has changed is the order of some codelists, not the codes themselves?

**Comment by odscjen** [comment]
have created an issue about the pre-commit problem https://github.com/Open-Telecoms-Data/open-fibre-data-standard/issues/285

**Comment by odscjen** [comment]
https://github.com/Open-Telecoms-Data/open-fibre-data-standard/pull/284 to fix linkcheck error, once that's merged into main, rebase this branch/merge main into this branch

File: codelists/closed/spanStatus.csv
Comment (duncandewhurst) [inline codelists/closed/spanStatus.csv:2]: 'proposed' should come before 'planned' as in `nodeStatus.csv`

**Review by duncandewhurst — CHANGES_REQUESTED** [review]
Fine to merge once my comment is resolved.

**Comment by odscjen** [comment]
thanks @duncandewhurst but can't merge until you officially approve :)

**Review by duncandewhurst — APPROVED** [review]

**Pull request merged**
